src/signals/incident/components/form/MapSelectors/types.ts

```diff
@@ -59,7 +59,19 @@ export interface WfsFilter {
 export interface DataLayerProps {
   featureTypes: FeatureType[]
   desktopView?: boolean
-  reportedLayer?: boolean
+}
+
+export interface Meta extends Record<string, unknown> {
+  name?: string
+  endpoint: string
+  featureTypes: FeatureType[]
+  language?: Record<string, string>
+  wfsFilter?: string
+  extraProperties?: string[]
```

Comment:
Is this type correct? Can't recall having seen `extraProperties` being an array of strings.

Reply:
No wait, this is correct.